standalone/Microsoft.Private.Intellisense/xmldocs/netcoreapp/1033/System.Collections.Immutable.xml

```diff
@@ -716,7 +716,7 @@ NuGet package: System.Collections.Immutable (about immutable collections and how
       <returns>An immutable array that contains the current contents of this <see cref="T:System.Collections.Immutable.ImmutableArray`1.Builder" />.</returns>
     </member>
     <member name="M:System.Collections.Immutable.ImmutableArray`1.CastArray``1">
-      <summary>Initializes a new instance of the <see cref="T:System.Collections.Immutable.ImmutableArray`1" /> struct by casting the underlying             array to an array of type <paramref name="TOther" />.</summary>
+      <summary>Initializes a new instance of the <see cref="T:System.Collections.Immutable.ImmutableArray`1" /> struct by casting the underlying array to an array of type .</summary>
```

Comment:
This looks like a regression.
https://docs.microsoft.com/en-us/dotnet/api/system.collections.immutable.immutablearray-1.castarray?view=netcore-3.0

![image](https://user-images.githubusercontent.com/6527137/64586058-e4710500-d34f-11e9-951f-0426f09f17e6.png)

https://github.com/dotnet/dotnet-api-docs/blob/9ea0edef8fe9034aee6b811dda7f6ee4ee9f55a7/xml/System.Collections.Immutable/ImmutableArray%601.xml#L311-L312

Reply:
Agreed. There's something weird with this build.

Reply:
Damn it! I used the wrong tag. Can we fix it here?

Reply:
```suggestion
      <summary>Initializes a new instance of the <see cref="T:System.Collections.Immutable.ImmutableArray`1" /> struct by casting the underlying array to an array of type <paramref name="TOther" />.</summary>
```